src/schedule/ll_schedule.c

```diff
@@ -353,6 +354,33 @@ static int schedule_ll_task(void *data, struct task *task, uint64_t start,
 
 	pdata->period = period;
 
+	/* for full synchronous domain, calculate ratio and initialize skip_cnt for task */
+	if (sch->domain->full_sync) {
+		pdata->ratio = 1;
+		pdata->skip_cnt = (uint16_t)(-1);
```

Comment:
It probably best to set a macro to 0xffff and have this as the default skip_cnt, which can then be used throughout the code.